### ISSUE TITLE
Support for measures

### ISSUE THREAD

**Issue by papillot** [molstar/mol-view-spec#20]
I would like to know if the specs support elements such as measures (distance, angle, dihedral)?

**Comment by JonStargaryen** [comment]
Hi @papillot,

We are considering this as one of the next steps. Thanks for your interest and suggestion. This is helpful to the dev team when it comes to prioritizing features.

Best,
Sebastian

**Comment by papillot** [comment]
Thanks for the heads-up, that's useful to know

**Comment by Glucksistemi** [comment]
Yes, it sounds like a feature I would use myself when it will be done

**Comment by dsehnal** [comment]
Measures are now supported.
